### PR TITLE
chore: remove ebpf from init containers

### PR DESCRIPTION
**Pull request by bartsmykla**
As there were some security considerations, we decided to
temporarily remove option to install transparent proxy using ebpf
in init containers

### Checklist prior to review

- [x] Link to docs PR or issue -- this PR removes unreleased feature
- [x] Link to UI issue or PR -- this PR removes unreleased feature
- [x] Is the [issue worked on linked][1]? -- none
- [x] The PR does not hardcode values that might break projects that depend on kuma (e.g. "kumahq" as a image registry) -- this PR removes unreleased feature
- [x] The PR will work for both Linux and Windows, system specific functions like `syscall.Mkfifo` have equivalent implementation on the other OS -- this PR removes unreleased feature
- [x] Unit Tests -- this PR removes unreleased feature
- [x] E2E Tests -- this PR removes unreleased feature
- [x] Manual Universal Tests -- 
- [x] Manual Kubernetes Tests --
- [x] Do you need to update [`UPGRADE.md`](/UPGRADE.md)? -- this PR removes unreleased feature
- [x] Does it need to be backported according to the [backporting policy](/CONTRIBUTING.md#backporting)? -- no

[1]: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

### CODE REVIEW

**Comment by slonka** [comment]
If this was a separate command we could probably do `cmd.hidden = true` - maybe something to remember in the future (if we can)

File: app/kumactl/cmd/install/testdata/install-control-plane.dump-values.yaml [hunk 1]
```diff
@@ -672,12 +672,6 @@ experimental:
   ebpf:
     # -- If true, ebpf will be used instead of using iptables to install/configure transparent proxy
     enabled: false
```
Comment (slonka) [inline app/kumactl/cmd/install/testdata/install-control-plane.dump-values.yaml:674]: do we need this section?

File: deployments/charts/kuma/values.yaml [hunk 2]
```diff
@@ -672,12 +672,6 @@ experimental:
   ebpf:
```
Comment (slonka) [inline deployments/charts/kuma/values.yaml:672]: same question as above

**Review by slonka — COMMENTED** [review]
LGTM, just 1 question.

**Review by slonka — APPROVED** [review]

File: app/kumactl/cmd/install/testdata/install-control-plane.dump-values.yaml (hunk 1 above)
Comment (bartsmykla) [reply app/kumactl/cmd/install/testdata/install-control-plane.dump-values.yaml:674]: Yes, it is used in cni/ebpf

File: deployments/charts/kuma/values.yaml (hunk 2 above)
Comment (bartsmykla) [reply deployments/charts/kuma/values.yaml:672]: We are using this value in cni/ebpf

**Pull request merged**